src/seedu/addressbook/Main.java

```diff
@@ -21,9 +21,13 @@ public static void main(String[] args) {
     }
 
     @Override
-    public void start(Stage primaryStage) throws Exception {
-        Gui gui = new Gui(new Logic(), VERSION);
-        gui.start(primaryStage, this);
+    public void start(Stage primaryStage) {
```

Comment:
Start() should still throw the Exception, or else the application will hang if the exception occurs

Reply:
this only occurs only startup. i've tested the app wont open at all

Reply:
Yea, but does the app close? I mean when you click run, then application won't open, but the run button in the IDE is is on, which implies that the app is hanging

Reply:
the runtimexception added will close the app

Reply:
But Codacy no like raw exceptions.
Just start throws Exception
 
Then catch (Exception e) {
   Log
    throw e;
}

Reply:
yeah sorry ill change it to a normal exception